seacrowd/sea_datasets/alorese/alorese_url.py

```diff
@@ -0,0 +1,710 @@
+_URLS_DICT = {
```

Comment:
Just curious: how were you able to generate this dictionary? Do you think it's possible to automate this process instead instead of keeping this file?

Reply:
Hi @ljvmiranda921  ! Based on the [archive](https://archive.mpi.nl/tla/islandora/object/tla:1839_1cd5563b_cfb4_4b0b_bc05_a18babddc86e?asOfDateTime=2019-03-01T10:32:30.790Z), here are the steps:

1. I scraped through the pagination to grab all of the blob url that leads to the [detail page](https://archive.mpi.nl/tla/islandora/object/tla%3A1839_77a7f739_3f6f_4ce4_b6ef_916e3825fa6a)
2. From each detail page, I scraped all of the `.wav` blob url like [this](https://archive.mpi.nl/tla/islandora/object/tla%3A1839_5db33109_854a_47ac_90c9_4e48a6d29643)
3. Now for the hard part, in order to match the `.wav` with the matching `.xml` (caption), I scraped it through [this](https://archive.mpi.nl/tla/islandora/object/tla%3A1839_1cd5563b_cfb4_4b0b_bc05_a18babddc86e?asOfDateTime=2019-03-01T10%3A32%3A30.790Z&f%5B0%5D=cmd.Format%3A%22text%5C/x%5C-eaf%5C%2Bxml%22) blob and find the mosts similar naming for each corresponding file with the `.wav` filename
4. Sadly, the name does not always have the same file naming as the `.wav` file, so I have to recheck one by one. Not to mention there might be `.wav` that does not have any `.xml`

From **4**, I reconsider to elicit my scraping script to the code since it might cause:
1. any trouble if slight UI change happens
2. slow process, pagination scraping of ~200 items, and it does not include the detail page scraping again + caption so the quickest is roughly ~200 * 3 = 600 URLs. Not to mention the time needed might vary more due to user's network speed difference.

Furthermore, I have gone through the discussion in the discord before implementing `alorese` cause this one is a little bit tricky. Got the approval to the such "bulky" method, but for the sake of better result. I can't really open my Discord now, will paste the discussion thread later on.

Reply:
Ah no worries! Thanks for outlining the steps! I guess there's no need to have this be automated, what you did is OK already. Perhaps we should document that process somewhere? Maybe as a docstring in the `alrose_url.py` file? Just a short description to ensure that future folks can replicate where the URLs are coming from. :)

Reply:
With pleasure! Please check the newest commit 😄.